pkg/volume/volume.go

```diff
@@ -132,11 +132,39 @@ type Attacher interface {
 	Attach() error
 }
 
```

Comment:
Hmm.. only one can be moved I think because GetPath() is applicable to all volumes not just attachable ones